packages/lesswrong/lib/vulcan-lib/utils.ts

```diff
@@ -286,7 +286,16 @@ export const sanitizeAllowedTags = [
   'ol', 'nl', 'li', 'b', 'i', 'u', 'strong', 'em', 'strike', 's',
   'code', 'hr', 'br', 'div', 'table', 'thead', 'caption',
   'tbody', 'tr', 'th', 'td', 'pre', 'img', 'figure', 'figcaption',
-  'section', 'span', 'sub', 'sup', 'ins', 'del', 'iframe'
+  'section', 'span', 'sub', 'sup', 'ins', 'del', 'iframe',
+  
+  //MathML elements (https://developer.mozilla.org/en-US/docs/Web/MathML/Element)
+  "math", "mi", "mn", "mo", "ms", "mspace", "mtext", "merror",
+  "mfrac", "mpadded", "mphantom", "mroot", "mrow", "msqrt", "mstyle",
+  "mmultiscripts", "mover", "mprescripts", "msub", "msubsup", "msup", "munder",
+  "munderover", "mtable", "mtd", "mtr",
+  
+  // MathML elements that are not included (because they're nonvisual or deprecated)
+  "maction", "semantics", "annotation", "annotation-xml", "menclose", "mfenced",
```

Comment:
Please remove maction: https://html5sec.org/#130

Reply:
Removed (along with all the other ones marked deprecated). I'm annoyed that MDN failed to document that this could include an `href`, which is hazardous. I checked and the sanitizer wouldn't have passed the href through (only whitelisted attribute names pass), so it wouldn't have been XSS, but it's a bit close to XSS for comfort.